crates/sncast/tests/data/multicall_configs/deploy_invoke_calldata_ids.toml

```diff
@@ -1,19 +1,19 @@
 [[call]]
 call_type = "deploy"
-class_hash = "0x7644be7f58307726aa836e945edede13e9e08c38eaf2186d4d48eca7dd435ac"
+class_hash = "0x51ce0a6fdc37672013380682f1d39871daea3acef92bc0891464aa55c858a45"
```

Comment:
How were these obtained?

Reply:
from some file called `.env... devnet...`
It was consulted with @THenry14